src/plugins/data_source_management/public/components/direct_query_data_sources_components/acceleration_management/acceleration_table.tsx

```diff
@@ -206,11 +206,11 @@ export const AccelerationTable = ({
   const tableActions = [
     {
       name: 'Query Data',
-      description: 'Query in Observability Logs',
+      description: 'Query in Discover',
       icon: 'discoverApp',
       type: 'icon',
       onClick: (acc: CachedAcceleration) => {
-        onDiscoverIconClick(acc, dataSourceName, application);
+        onDiscoverIconClick(acc, dataSourceName, dataSourceMDSId, application);
```

Comment:
With a local cluster dataSourceMDSId would be undefined here and later in the actual function that does the redirection it will pass it in as '', i.e. within rison encoding format